### PR TITLE
try github actions

### PR DESCRIPTION
**Pull request by simonbyrne**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ jobs:
 
     env:
       MPICH_INTERFACE_HOSTNAME: localhost
+      JULIA_MPIEXEC_TEST_ARGS: "-hosts localhost"
```
Comment (simonbyrne) [inline .github/workflows/tests.yml:21]: @jakebolewski this seems to make it work

Comment (jakebolewski) [reply .github/workflows/tests.yml:21]: ok good to know, we'll have to have to add something similar to clima

Comment (jakebolewski) [reply .github/workflows/tests.yml:21]: probably needed both to avoid the fallback behavior :-/

**Comment by jakebolewski** [comment]
@simonbyrne I used `-host` (part of the standard) and not `-hosts`, not sure if that matters

**Comment by simonbyrne** [comment]
ah, good to know. This wasn't the cause, but I updated it.

**Pull request merged**